rtloader/common/rtloader_mem.c

```diff
@@ -15,9 +15,27 @@ static rtloader_free_t rt_free = free;
 static cb_memory_tracker_t cb_memory_tracker = NULL;
 
 void _set_memory_tracker_cb(cb_memory_tracker_t cb) {
+
+    // TODO: we build with gcc on windows so the __sync_synchronize
```

Comment:
This indentation and the other `TODO` down below is weird.

Reply:
It's the clang-format indentation, but I'm going to remove this as this should work on windows, so it'll no longer be a sore on the eye